specification/monitor/resource-manager/readme.md

```diff
@@ -27,16 +27,26 @@ These are the global settings for the MonitorClient API.
 ``` yaml !$(python) || !$(track2)
 title: MonitorClient
 ```
+
 ``` yaml $(python) && $(track2)
 title: MonitorManagementClient
 ```
 
 ``` yaml
 description: Monitor Management Client
 openapi-type: arm
-tag: package-2020-03
+tag: package-2016-06
 ```
 
+
+### Tag: package-2016-06
+
+These settings apply only when `--tag=package-2016-06` is specified on the command line.
+
+```yaml $(tag) == 'package-2016-06'
```

Comment:
```suggestion
### Tag: package-2016-06-only

These settings apply only when `--tag=package-2016-06-only` is specified on the command line.

```yaml $(tag) == 'package-2016-06-only'
```

Reply:
Thanks for reviewing, I've incorporated your suggestion.